src/controller/reco_controller.py

```diff
@@ -141,12 +145,12 @@ def get_model_type(self):
     #
     # TODO - refactor this into a factory class or similar
     #
-    def get_item_viewer(self, item_dto: ItemDto):
+    def get_item_viewer(self, item_dto: ItemDto, app_explorer_instance: "RecoExplorerApp"=None):
```

Comment:
Das hier müsste ein Union Type sein:
```suggestion
    def get_item_viewer(self, item_dto: ItemDto, app_explorer_instance: RecoExplorerApp | None =None):
```